baseplate/clients/thrift.py

```diff
@@ -69,6 +70,21 @@ class ThriftContextFactory(ContextFactory):
 
     """
 
+    PROM_PREFIX = "bp_thrift_pool"
+    PROM_LABELS = ["client_cls"]
+
+    prom_total_connections = Gauge(
+        f"{PROM_PREFIX}_size",
```

Comment:
You were going to change this to total right?

Reply:
`total` has a special meaning in Prometheus to denote counters, changed it to `connections` so it has a unit.